tests/CMakeLists.txt

```diff
@@ -1,12 +1,18 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.8)
 
-project(GSLTests CXX)
+project(GSLTests LANGUAGES CXX)
 enable_testing()  # again, for support standalone testing
 
+set(GSL_CXX_STANDARD "14" CACHE STRING "Use c++ standard")
+
+set(CMAKE_CXX_STANDARD ${GSL_CXX_STANDARD})
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CXX_STANDARD_REQUIRED ON)
+
```

Comment:
This is fundamentally where setting the `GSL_CXX_STANDARD` actually matters. Setting it via GSL just complicates things.

Reply:
Also to avoid writing complicated CMake just raise the cmake_minimum, since it won't impact clients at all.